src/TagInput/driver.js

```diff
@@ -1,53 +1,196 @@
-import InputComponentDriver
-    from '../Testing/CommonDrivers/inputComponentDriver';
+import { Tag } from '../index';
 
-export default class TagInputDriver extends InputComponentDriver
+const ERR = {
+    TAGINPUT_ERR : ( event, state ) =>
+        `TagInput cannot simulate ${event} since it is ${state}`,
+};
+
+export default class TagInputDriver
 {
     constructor( wrapper )
     {
-        super( wrapper );
         this.wrapper = wrapper;
+        this.control = wrapper.find( 'InputContainer' );
     }
 
     clickCloseTagByIndex( index = 0 )
```

Comment:
TagInput’s `clickCloseTagByIndex( index )` and
`clickCloseTagByLabel( label )` should be simply one method: `onClickClose( index )`